engine.py

```diff
@@ -87,24 +88,30 @@ def pre_app_init(self):
         self._tk_vred = self.import_module("tk_vred")
 
         # check for version compatibility
-        vred_version = int(vrController.getVredVersionYear())
-        self.logger.debug("Running VRED version {}".format(vred_version))
-        if vred_version > self.get_setting("compatibility_dialog_min_version", 2021):
+        self.vred_version = os.getenv("TK_VRED_VERSION", None)
+        self.logger.debug("Running VRED version {}".format(self.vred_version))
+        if (
+            self._version_check(
+                self.vred_version,
+                str(self.get_setting("compatibility_dialog_min_version")),
+            )
+            >= 0
```

Comment:
I think we want this condition to be `>0` instead of `>=0`.

`>=0` will cause the warning if the version is greater than the given version, but also if it is equal to it.